README.md

```diff
@@ -27,10 +28,9 @@ an RBAC ruleset, an add-on, an application, etc...
 
 ## Arlon Benefits
 
-- Unifies infrastructure and application management
-- Improves time to market by enabling better velocity for developers through infrastructure management that is more fluid and agile  
-- Reduces the risk of unexpected infrastructure & application downtime and outages - with consistent management of infrastructure and applications  
-- Allows IT and Platform Ops admins to operate large scale of clusters & applications with significantly reduced team size & operational overhead 
+- Improves time to market by enabling better velocity for developers through infrastructure management that is more fluid and agile. Define, store, change and enforce your cluster infrastructure at scale.  
+- Reduces the risk of unexpected infrastructure downtime and outages, or unexpected security misconfiguration, with consistent management of infrastructure and security policies.   
+- Allows IT and Platform Ops admins to operate large scale of clusters & infrastructure with significantly reduced team size & operational overhead, using GitOps. 
```

Comment:
Same comment: find a way to incorporate the application angle.

Reply:
Good points. Updated. Take a look